### PR TITLE
Rede fantasy

### PR DESCRIPTION
**Pull request by felipepasc**
## Type of change

- [ X] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [X ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [X ] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X ] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/fantasy/manifest.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  ],
+  "gamemodes": {
+    "prison": {
+      "name": "Prison Phoenix",
+      "color": "#FFAA00",
+      "command": "/serverconnector prison-phoenix"
+    }
+  },
+  "social": {
+    "discord": "https://discord.redefantasy.com",
```
Comment (DoJapHD) [inline minecraft_servers/fantasy/manifest.json:16]: Discord has to be https://discord.gg/[INVITE]

File: minecraft_servers/fantasy/manifest.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "web_shop": "https://loja.redefantasy.com",
```
Comment (DoJapHD) [inline minecraft_servers/fantasy/manifest.json:17]: Website is just standart nginx-starting website

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/fantasy/manifest.json (hunk 1 above)
Comment (felipepasc) [reply minecraft_servers/fantasy/manifest.json]: Resolved

**Comment by felipepasc** [comment]
Resolved

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**